.gitlab-ci.yml

```diff
@@ -0,0 +1,173 @@
+stages:
+  - rc # 'release mode'; stage will be autouploaded to bintray in future
+  - debug
+
+.artifacts: &artifacts
+  artifacts:
+    paths:
+      - build/
+
+.cache: &cache
+  cache:
+    key: "$CI_BUILD_NAME"
+    paths:
+      - cache/
+
+
+#================================ DEBIAN-BASED ================================
+
+.build_rc_package_deb: &build_rc_package_deb
+  stage: rc
+  script:
+    - mkdir build || true # allowed to fail
+    - cd build
+    - cmake .. -DWITH_SERVER=1 -DCMAKE_BUILD_TYPE=Release -DPACKAGE_GENERATOR=DEB
```

Comment:
Can you update this to use `-DCPACK_GENERATOR` instead of `-DPACKAGE_GENERATOR` as implemented in #2373? Thank you